rules/framework.bzl

```
@@ -70,14 +70,15 @@ def apple_framework(name, apple_library = apple_library, **kwargs):
 
     library = apple_library(name = name, **kwargs)
     framework_deps = []
+    testonly = kwargs.pop("testonly", False)
```

I find it kind awkward to pass it explicitly everywhere but here:
```suggestion
    testonly = kwargs.pop("testonly", False)

    library = apple_library(name = name, testonly = testonly, **kwargs)
    framework_deps = []
```